sources/academy/webscraping/puppeteer_playwright/common_use_cases/paginating_through_results.md

```diff
@@ -80,36 +72,35 @@ await browser.close();
 import puppeteer from 'puppeteer';
 
 const repositories = [];
-
 const BASE_URL = 'https://github.com';
 const REPOSITORIES_URL = `${BASE_URL}/orgs/facebook/repositories`;
 
 const browser = await puppeteer.launch({ headless: false });
-const page = await browser.newPage();
-
-await page.goto(REPOSITORIES_URL);
-
-const lastPageLabel = await page.$eval('a[aria-label*="Page "]:nth-last-child(2)', (elem) => elem.getAttribute('aria-label'));
-// This will output 4
-const lastPage = +lastPageLabel.replace(/\D/g, '');
+const firstPage = await browser.newPage();
+await firstPage.goto(REPOSITORIES_URL);
 
-console.log(lastPage);
+const lastPageLabel = await firstPage.$eval(
+    'a[aria-label*="Page "]:nth-last-child(2)',
+    (element) => element.getAttribute('aria-label'),
+);
+const lastPageNumber = Number(lastPageLabel.replace(/\D/g, ''));
+console.log(lastPageNumber);
 
 await browser.close();
 ```
 
 </TabItem>
 </Tabs>
 
-> [Learn more](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-last-child) about the `:nth-last-child` pseudo-class. It works similar to `:nth-child`, but starts from the bottom of the parent element's children instead of from the top.
+:::tip :nth-last-child
```

Comment:
Just talked about it on a call, but I'll summarize:

- I don't have a strong opinion, I follow @TC-MO's advice on including the titles.
- It's some work to think about the title, but when I look at the result, I think it's for the better. I feel the page is easy to scan. It's clearer what the colorful box is about and whether I want to read it or not.
- Sometimes it helps with writing too, because thinking about "how do I call this box" resulted in "the box is a mess, I must do this in a different way". Similar to coding, when thinking about "how do I name this function" results in splitting the function to three different ones.

Thus I think I do like the titles.